cli/src/main/java/com/box/l10n/mojito/cli/command/extractiondiffnotifier/ExtractionDiffNotifierGithub.java

```diff
@@ -16,6 +16,9 @@ public class ExtractionDiffNotifierGithub implements ExtractionDiffNotifier {
   /** logger */
   static Logger logger = getLogger(ExtractionDiffNotifierGithub.class);
 
+  public static String COMMENT_REGEX =
+      ".*[\\d]+ string[s]{0,1} removed and [\\d]+ string[s]{0,1} added.*";
```

Comment:
The extraction diff notification message is configurable so this regex should be configurable too